Test.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <stdexcept>
+
+class FactorialCalculator {
+private:
+    
+public:
+    // Method to calculate the factorial of a number
+    static long long calculate(long long number) {
+        if (number < 0) {
+            throw std::invalid_argument("Number must be non-negative.");
+        }
+
```

Comment:
Consider adding comments to explain the purpose and functionality of this class and its methods. This will improve readability and maintainability.